archunit/src/main/java/com/tngtech/archunit/lang/ConditionEvents.java

```diff
@@ -36,12 +38,26 @@ public ConditionEvents() {
     }
 
     private final Multimap<Type, ConditionEvent> eventsByViolation = ArrayListMultimap.create();
+    private Optional<String> informationAboutNumberOfViolations = Optional.absent();
```

Comment:
As far as I can judge `informationThatViolationsHaveBeenLimited` would be a better description.

Reply:
But it is more generic, right? You don't have to specify "the information has been limited" you could also say "the heuristically determined number of violations reported does not match my custom violations, I'll count and specify the number of violations myself". I'm just not using that within ArchUnit itself, but this is public API that users of ArchUnit could use themselves...

Reply:
I agree.